test/uk/gov/hmrc/cgtpropertydisposals/service/returns/transformers/ReturnTransformerServiceImplSpec.scala

```diff
@@ -675,35 +675,6 @@ class ReturnTransformerServiceImplSpec extends WordSpec with Matchers with MockF
           result.map(_.exemptionsAndLossesDetails.annualExemptAmount) shouldBe Right(AmountInPence(1234L))
         }
 
-        "there is a net loss" in {
```

Comment:
we will need to read back these values eventually, but we can do that when `CompleteReturn` handles both single disposals and multiple disposals and also calculated and non-calulated journeys